client/jetpack-cloud/sections/onboarding-tours/add-new-site-tour-step-2/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslate } from 'i18n-calypso';
+import { useContext } from 'react';
+import GuidedTour from 'calypso/jetpack-cloud/components/guided-tour';
+import { useSelector } from 'calypso/state';
+import { items } from 'calypso/state/notices/reducer';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import SitesOverviewContext from '../../agency-dashboard/sites-overview/context';
+
+export default function AddNewSiteTourStep2() {
+	const translate = useTranslate();
+	const { mostRecentConnectedSite } = useContext( SitesOverviewContext );
+	const hasAddNewSiteTourPreference = useSelector( ( state ) =>
+		getPreference( state, 'jetpack-cloud-site-dashboard-add-new-site-tour-step-1' )
```

Comment:
Is it possible to move the preference name to a constant file? This is because it could potentially be altered elsewhere without updating it here.

Reply:
Added in e2ae741